src/static/report_a_problem.html

```diff
@@ -32,10 +32,17 @@
     const { shell, ipcRenderer } = require('electron');
     const { getLogger } = require('../logger');
     const logger = getLogger(__filename.split('/').slice(-1)[0]);
-    ipcRenderer.on('variable-reply', (event, arg) => {
-      userDataPath = arg.userDataPath;
+
+    let userDataPath;
+
+    ipcRenderer.invoke('user-data')
+    .then(response => {
+      userDataPath = response;
+    })
+    .catch(e => {
+        logger.error("Error processing user data path.");
```

Comment:
Same deal as the about.html here.

Reply:
Same: https://github.com/natcap/invest-workbench/pull/135/commits/6be3fcc4ec592f9cc8366d654a464439f8b1e245